### ISSUE TITLE
Instance segmentation in COCO format

### ISSUE THREAD

**Issue by Robotatron** [Unity-Technologies/PeopleSansPeople#10]
Great project, glad I found it, very helpful! I am wondering if you could point me into the right direction regarding converting instance segmentation masks into the COCO format?

In your README it says: 

> Generated image and corresponding labels: 2D bounding box, human keypoints, semantic **and instance segmentation masks in COCO format**. 

However in your repo I did not find scripts to convert Unity Perception instance segmentation data to the COCO format. I thought it was done within Unity, however again, briefly skimming in the /Scripts folder I did not found anything related to that.

Whether Unity Perception nor Data Insights python package provide functionality to convert instance segmentation masks to the COCO format. 
Any advice or help is appreciated! Thanks.

**Comment by Robotatron** [comment]
Yes, I saw your PR and it was just merged into datainsights. Thank you, I will test it out!

**Comment by Robotatron** [comment]
> @Robotatron datasetinsights now support instance segmentation. Let us know if you have any other questions.

Managed to test it and run into an issue. I've generated a dataset with Unity Perception and converted to the COCO format with your merged PR. 

When loading the generated COCO annotations with pycocotools COCO class, only subset of annotations are present. Out of 20 images generated in Unity, only 2 images have annotations in the converted COCO format in **coco.anns** list:
![image](https://user-images.githubusercontent.com/95919873/179777556-e7e956a7-5639-4c98-88cb-85cd1d9dcc1b.png)
![image](https://user-images.githubusercontent.com/95919873/179777612-346f9083-3ed0-486d-92e6-0efeff740b20.png)

And if one inspects **imgToAnns** of the COCO class, a dictionary that maps img ids to annotations ids it shows all 8 images having the same annotations:
![image](https://user-images.githubusercontent.com/95919873/179778273-d4a6bc8b-6f37-4305-827d-5faf546238f4.png)

I then plotted the JSON COCO file directly, without using the COCO class from pycocoutils and everything was fine. 

I am not really sure what happens, but it seems the COCO class imports the data incorrectly, which is weird, since pycocotools are widely used.
